dto/pom.xml

```diff
@@ -38,5 +38,9 @@
       <groupId>cisd</groupId>
       <artifactId>jhdf5</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
```

Comment:
Is this needed for something? I tried ripping it out of all 3 poms and the Swagger docs still work fine